modules/jupyter-widget/src/playground/create-deck.js

```diff
@@ -196,9 +198,24 @@ function createDeck({
   tooltip,
   handleEvent,
   customLibraries,
-  configuration
+  configuration,
+  showError,
 }) {
   let deckgl;
+  const onError = (e) => {
+    if (showError) {
+      const uiErrorText = window.document.createElement('pre');
+      uiErrorText.textContent = `Error: ${e.message}\nSource: ${e.source}\nLine: ${e.lineno}:${e.colno}\n${e.error ? e.error.stack : ''}`;
+      uiErrorText.className = 'error_text';
+
+      container.appendChild(uiErrorText);
```

Comment:
Perhaps worth still logging the error to the console as it is then easier to navigate, e.g. you can click through to the file where it originated from

Reply:
I agree.  Fixed.